package.json

```diff
@@ -70,6 +70,12 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "prettier": {
+    "printWidth": 100,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "all",
+    "useTabs": true,
+    "tabWidth": 2,
```

Comment:
Moved Prettier rules to package.json as prettierrc was getting ignored, causing unnecessary Git changes
LMK if there's an alternate preferred way to do this, but Platform UI is set up w/ rules in package.json

Reply:
personally I prefer having separate files for separate concerns...

The manifest file (package.json) tends to get quite busy as it is, in most projects.

In case it helps, `prettier.config.js` works fairly well in Arranger. https://github.com/overture-stack/arranger/blob/develop/prettier.config.js

Reply:
Fixed in: https://github.com/icgc-argo/rdpc-ui/pull/102